core/arch/arm/kernel/thread_spmc.c

```diff
@@ -1517,7 +1517,7 @@ static void handle_notification_get(struct thread_smc_args *args)
 		w2 = 0;
 		goto out;
 	}
-	vm_id = FFA_SRC(args->a1);
+	vm_id = FFA_DST(args->a1);
```

Comment:
Use of macro `FFA_DST()` looks a bit strange as this command ABI does not use SRC/DST in respectively upper/lower 16bit part of the value passed . An explicit 16bit LSB extract instruction would look more appropriate.

Reply:
This follows the same logic as in for instance `handle_notification_unbind()` and `handle_notification_bind()` above.
Would it help if I added the comment `/* The destination of the notification */`?

Reply:
Ok, makes sense. Discard my comment.
`Reviewed-by: Etienne Carriere <etienne.carriere@foss.st.com>`